Write new task: Configuring a Pod to Use a Volume for Storage

steveperry-53 opened the pull request:
<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1875)
<!-- Reviewable:end -->

pwittrock commented:
I think it is missleading to call an emptydir volume persistent storage.  It disappears when the Pod is deleted.  I think most people would consider Persistent Storage something like a Persistent Disk that has its own lifetime outside of the Pod / Node.

steveperry-53 commented:
@pwittrock I see your point about not calling an emptyDir persistent storage. Some of our existing docs do call this persistent storage, for example http://kubernetes.io/docs/user-guide/production-pods/. But maybe we should come up with another title.
Configuring a Pod to Use a Volume for Storage
Using a Volume to Save State Across Container Restarts
Other ideas?

pwittrock commented:
Hm, not sure.  @saad-ali?  "Configuring a Pod to Use a Volume for Storage" sounds good

saad-ali commented:
I agree with @pwittrock. Persistent storage is inaccurate. `Configuring a Pod to Use a Volume for Storage` sounds correct since it does not imply the nature of the volume (which may be ephemeral or persistent).  Existing docs should be corrected as well. @steveperry-53 do you mind making the change or opening an bug to make sure it is fixed.

steveperry-53 commented:
Removed "persistent" from this new topic.
Also removed "persistent" from existing topic in https://github.com/kubernetes/kubernetes.github.io/pull/1931.

steveperry-53 commented:
@pwittrock, OK to merge?

pwittrock commented on docs/tasks/configure-pod-container/configure-volume-storage.md:
```diff
@@ -0,0 +1,107 @@
+
+A Container's file system lives only as long as the Container does, so when a
+Container terminates and restarts, changes to the filesystem are lost. For more
+consistent storage that lasts for the life of a Pod, you can use a
```
nit: 

> For more consistent storage that lasts independent of the container, you can use a...

steveperry-53 replied:
Done.

pwittrock commented on docs/tasks/configure-pod-container/configure-volume-storage.md:
```diff
@@ -0,0 +1,107 @@
+### Configuring a volume for a Pod
+
+In this exercise, you create a Pod that runs one Container. The Pod has a
+volume that lasts for the life of the Pod, even if the container terminates and
```
nit: Maybe

> This Pod has a volume that lasts ...

pwittrock commented on docs/tasks/configure-pod-container/configure-volume-storage.md:
```diff
@@ -0,0 +1,107 @@
+
+In this exercise, you create a Pod that runs one Container. The Pod has a
+volume that lasts for the life of the Pod, even if the container terminates and
+restarts. Here is the configuration file for the Pod:
```
WDYT of:

In this exercise, you create a Pod that runs one Container. The Pod has an` Volume of type EmptyDir`, which lasts for the life of the Pod, even if the container terminates and restarts.  `Some Volumes, such as PersistentVolumeClaims, even continue to exist outside the lifetime of the Pod.`

pwittrock commented:
Looks good.  Left a couple minor comments.  Merge at your discretion.
